src/git.js

```diff
@@ -123,7 +122,7 @@ class Git {
 	async createPrBranch() {
 		const prefix = BRANCH_PREFIX.replace('SOURCE_REPO_NAME', GITHUB_REPOSITORY.split('/')[1])
 
-		let newBranch = path.join(prefix, this.repo.branch)
+		let newBranch = path.join(prefix, this.repo.branch).replace(/\\/g, '/')
```

Comment:
I was having problems on my Windows machine as path.join generates the branch name with `\` instead of `/` and this char is not valid on a branch name. This change made it compatible with windows runners